"azurerm_kubernetes_cluster" - supports property "dns_prefix_private_cluster" (#11321)
terraform-providers/terraform-provider-azurerm: website/docs/r/kubernetes_cluster.html.markdown

* `default_node_pool` - (Required) A `default_node_pool` block as defined below.

* `dns_prefix` - (Optional) DNS prefix specified when creating the managed cluster. Changing this forces a new resource to be created.

> * `private_cluster_custom_dns_prefix` - (Optional) Specifies the Custom DNS prefix when creating the private cluster. Changing this forces a new resource to be created.

Reviewer: grammar and i think custom is redundant?

```suggestion
* `private_cluster_dns_prefix` - (Optional) Specifies the DNS prefix to use with private clusters. Changing this forces a new resource to be created.
```

also given there is a dns_prefix property, maybe this makes sense to keep them ordered/auto completed at the same time

```suggestion
* `dns_prefix_private_cluster` - (Optional) Specifies the DNS prefix to use with private clusters. Changing this forces a new resource to be created.
```
